extensions/provisioninginput/config.go

```diff
@@ -224,3 +228,7 @@ type Config struct {
 	ClusterSSHTests        []SSHTestCase                            `json:"clusterSSHTests,omitempty" yaml:"clusterSSHTests,omitempty"`
 	CRIDockerd             bool                                     `json:"criDockerd,omitempty" yaml:"criDockerd,omitempty"`
 }
+
+type RKE1CustomClusterDockerInstall struct {
+	InstallDockerURL string `json:"installDockerURL" yaml:"installDockerURL"`
```

Comment:
Rishab both Caleb and I meant have this be an attribute/entry for `provisioningInput` right @slickwarren

Reply:
Which means I don't think we would need `GetInstallDockerURL`, since customCluster already has access to `provisioningInput`

Reply:
correct

Reply:
Okay I've made the required changes and updated the PR description for yaml fields. Now we can pass the docker install url from provisioningInput. Once this change is approved, i'll backport and forward port it.